sample/src/androidTest/java/com/schibsted/spain/barista/sample/RecyclerViewInsideViewPagerTest.java

```diff
@@ -0,0 +1,40 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.BaristaAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.BaristaRecyclerViewActions.clickRecyclerViewItem;
+import static com.schibsted.spain.barista.BaristaRecyclerViewActions.clickRecyclerViewItemChild;
+import static com.schibsted.spain.barista.BaristaRecyclerViewActions.scrollTo;
+
+@RunWith(AndroidJUnit4.class) public class RecyclerViewInsideViewPagerTest {
```

Comment:
I just installed AndroidSquare codestyle in my machine and it seems that it formats like this. But let me move to the previous version... Actually, it was hard to me to find the `public class` statement some minutes ago :·/